### PR TITLE
Replace devicons font with inline icons

### PR DESCRIPTION
**Pull request by TheDancingCode**
I extracted svg versions of the github and twitter icons in the footer from the devicons icon font. It seemed a bit excessive to load a 203 KB font, only to use these two glyphs. The visual appearance is (near) identical, but you save two network requests and between 203 KB and 792 KB of font files.

### CODE REVIEW

File: views/partials/footer.html [hunk 1]
```diff
@@ -13,8 +13,18 @@
     </nav>
 
     <span class="footer-social">
-      <a class="footer-nav-item icon" href="https://github.com/electron" title="Github Organization"><i class="devicon-github-plain vertical-middle"></i></a>
-      <a class="footer-nav-item icon" href="https://twitter.com/electronjs" title="@electronjs on Twitter"><i class="devicon-twitter-plain vertical-middle"></i></a>
+      <a class="footer-nav-item" href="https://github.com/electron" aria-label="Github Organization">
+        <svg class="footer-social-icon icon-github" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```
[the remark this answers is not in the document]
Comment (TheDancingCode) [reply views/partials/footer.html]: The `alt` attribute is not recommended for svg images. Instead, I used the `<title>` element in the svg. Quoting tip n. 3 of ['Tips for Creating Accesible SVG'](https://www.sitepoint.com/tips-accessible-svg/) on Sitepoint:
> To put it simply, the `<title>` element provides an alternative text description of the SVG content. Think of it in the same way you would the `alt` attribute on the HTML  element.

[the remark this answers is not in the document]
Comment (ckundo) [reply views/partials/footer.html]: Hi friends, I help build @accesslint. I'm taking a peek at this and will post notes soon. cc @zeke.

[the remark this answers is not in the document]
Comment (ckundo) [reply views/partials/footer.html]: @TheDancingCode you're right, AccessLint posts a comment on missing `alt` for `role='img'`, but doesn't exclude SVG. `<title>` generates the correct accessible name. Thanks for pointing that out.

File: views/partials/footer.html [hunk 2]
```diff
@@ -13,8 +13,18 @@
     </nav>
 
     <span class="footer-social">
-      <a class="footer-nav-item icon" href="https://github.com/electron" title="Github Organization"><i class="devicon-github-plain vertical-middle"></i></a>
-      <a class="footer-nav-item icon" href="https://twitter.com/electronjs" title="@electronjs on Twitter"><i class="devicon-twitter-plain vertical-middle"></i></a>
+      <a class="footer-nav-item" href="https://github.com/electron" aria-label="Github Organization">
```
Comment (ckundo) [inline views/partials/footer.html]: Quick note while I'm here, I noticed the `aria-label` on this link overrides the accessible name defined by the `<title>` in the SVG.

**Comment by vanessayuenn** [comment]
👋  @TheDancingCode, thank you for opening this PR! While shaving off the devicons font loading would be beneficial in terms of performance, it hampers maintainability and makes it more difficult to add glyphs. electronjs.org is still under active development at the moment, and we will certainly utilize devicon more extensively in near future. Thus, we decide we won't be merging this PR at the moment. I am going to close this PR for now, but please feel free to continue the discussion here if you have further comments or concerns.

**Pull request closed without merging**